web/crux/prisma/migrations/20220727115314_add_note_type/migration.sql

```diff
@@ -0,0 +1,5 @@
+-- CreateEnum
+CREATE TYPE "NodeTypeEnum" AS ENUM ('dagent', 'crane');
```

Comment:
You used everywhere docker and k8s, it would be more straightforward to use the same values here, not? Open for discussion.

Reply:
Like the naming idea. It's okay for me.

Reply:
+1